db/db.py

```diff
@@ -113,10 +113,10 @@ def close_db(self):
 for i in range(num_req):
 
 	QUERY_SELECT_JOIN = """
-	SELECT book_record.id, book_record.record_id, book_record.field_id, book_field.field_name
+	SELECT book_record.id, book_record.record_id, book_field.field_name
 	FROM book_record
 	INNER JOIN book_field
-	ON book_record.field_id=book_field.id and field_name="field5";
+	ON book_record.field_id=book_field.id and book_record.value_txt="field5";
```

Comment:
field "book_record.value_txt" has values ​​that are the same as "book_field.field_name" and I wanted to make a condition on a smaller table book_field.